app/models/community-solutions-analysis.ts

```
@@ -0,0 +1,124 @@
+      label: 'downvotes',
+      value: this.scoredSolutionDownvotesCount.toString(),
+      color: this.calculateColorUsingInverseThresholds(this.scoredSolutionDownvotesCount, downvotesCountThresholds),
+      explanationMarkdown: downvotesCountExplanationMarkdown,
+    };
+  }
+
+  calculateColorUsingInverseThresholds(value: number, thresholds: { green?: number; yellow?: number; red?: number }): 'green' | 'yellow' | 'red' {
+    if (thresholds.green !== undefined && value <= thresholds.green) return 'green';
+    if (thresholds.yellow !== undefined && value <= thresholds.yellow) return 'yellow';
+    if (thresholds.red !== undefined && value >= thresholds.red) return 'red';
+    return 'yellow';
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)